### ISSUE TITLE
Notify when someone signs up

### ISSUE THREAD

**Issue by YoruNoHikage** [indie-collective/community#34]
**Is your feature request related to a problem? Please describe.**
Could be nice to know when someone registered on Community.

**Describe the solution you'd like**
It could be a simple email or a discord text.

**Describe alternatives you've considered**
- Endlessly querying the database
- Having a member list on community

**Comment by engleek** [comment]
Probably easier to use a web hook to JMJ, or directly call the Discord API.

**Comment by YoruNoHikage** [comment]
Just re-discovered how Discord's webhooks work, just a simple fetch and we're good.

**Comment by engleek** [comment]
Resurrecting the issue. Do we still want this? Would be the right time, but we're linked to Discord now.

**Comment by YoruNoHikage** [comment]
Yup, just a simple fetch to add there https://github.com/indie-collective/community/blob/main/app/utils/auth.server.js#L138
